### PR TITLE
Completed button

### PR DESCRIPTION
**Pull request by nishasen**
Buttons

### CODE REVIEW

File: Stylesheet/Components/badge.css [hunk 1]
```diff
@@ -9,24 +9,26 @@
...
     border-radius: 10%;
+    border-bottom-right-radius: 40%;
     position: absolute;
     z-index: 9999;
     bottom: 0;
-    right: 0;
+    right: 0rem;
     border: 2px solid var(--st-white);
-}
+} */
```
Comment (Mohasin-Haque) [inline Stylesheet/Components/badge.css]: remove the commented code

File: Stylesheet/Components/badge.css [hunk 2]
```diff
@@ -9,24 +9,26 @@
     position: absolute;
     z-index: 9999;
```
Comment (Mohasin-Haque) [inline Stylesheet/Components/badge.css]: giving the higher z-index value may cause problem

**Review by Mohasin-Haque — COMMENTED** [review]
Your code looks good. But needs to work on the commented part. You may proceed!!

File: playground.html [hunk 3]
```diff
@@ -10,17 +10,21 @@
     <style>
         body {
             display: flex;
```
Comment (tanishq20) [inline playground.html:12]: I think it's not a good practice to use flex on the body.
Correct me if I'm wrong.

File: Stylesheet/Components/button.css [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+    align-items: center;   
+}
+
+.btn-over-image {
+    color: white;
+    font-size: 1.5rem;
+    font-weight: bold;
+    max-width: 9rem;
+    height: 5rem;
+    z-index: 9999;
```
Comment (tanishq20) [inline Stylesheet/Components/button.css:200]: Use nominal z-index value.

File: playground.html (hunk 3 above)
Comment (nishasen) [reply playground.html:12]: Actually it's just a temporary thing, to visualize my components well, that is not going to be added

File: Stylesheet/Components/button.css (hunk 4 above)
Comment (nishasen) [reply Stylesheet/Components/button.css:200]: Okay, will change that

File: Stylesheet/Components/badge.css (hunk 2 above)
Comment (nishasen) [reply Stylesheet/Components/badge.css]: Yes changing it

File: Stylesheet/Components/badge.css (hunk 1 above)
Comment (nishasen) [reply Stylesheet/Components/badge.css]: Yes doing that

**Pull request merged**